app/ts/App.tsx

```diff
@@ -375,11 +326,11 @@ export function App() {
 						<p>Balance: { bigintToDecimalStringWithUnknown(totalBalance.deepValue, 18n, 2) } REP / { bigintToDecimalStringWithUnknown(requiredBalance.deepValue, 18n, 2) } REP ({ fundedPercentageString.value } funded)</p>
 					</div>
 					<button class = { 'button button-secondary button-small' } onClick = { forceRefresh }> Refresh </button>
-					<Allowances maybeWriteClient = { maybeWriteClient } allowedRep = { allowedRep } handleUnexpectedError = { handleUnexpectedError }/>
 				</div>
 				<div class = 'form-group'>
 					<h3>Deposit REP</h3>
 					<p>Deposit REP to contract for Micah to withdawn. Micah is intended to use the REP to fund Augur v2 fork.</p>
```

Comment:
I feel like we should use the same text here.  Maybe this should be turned into a little component that we can reuse?
```suggestion
					<p>Deposit REP to a contract that Micah can withdraw from once { bigintToDecimalStringWithUnknown(requiredBalance.deepValue, 18n, 2) } REP is reached. Micah commits (via a gentleman's agreement) to use all of the REP to fund an Augur v2 fork.</p>
```

Reply:
this is the same line you are changing, this is in older base of main :D